sc-server-auth/tests/common.py

```diff
@@ -0,0 +1,38 @@
+from main import app
+from fastapi.testclient import TestClient
+from config import params, BASE_AUTH_SERVER_URL
+from server import constants as cnt
+import unittest
+
+test_client = TestClient(app)
+get_tokens_url = BASE_AUTH_SERVER_URL + params[cnt.GET_TOKENS_ENDPOINT]
+get_access_token_url = BASE_AUTH_SERVER_URL + params[cnt.GET_ACCESS_TOKEN_ENDPOINT]
+user_url = BASE_AUTH_SERVER_URL + params[cnt.USER_ENDPOINT]
+users_url = BASE_AUTH_SERVER_URL + params[cnt.USERS_ENDPOINT]
+
+
+def create_user_request(token="wrong_token",
```

Comment:
Try to follow a unique writing codestyle. I think this part should looks like a _get_token_ function.

Reply:
These functions works like factories for creating requests. For each type of request its own factory function. And I named functions according goal of request and endpoint url.
Can you explain your point of view about your variant functions naming  and comment on my variant?